armotypes/clusters.go

```diff
@@ -3,19 +3,21 @@ package armotypes
 import "time"
 
 type ClusterInfo struct {
-	Cluster        string       `json:"cluster"`
-	NodeCount      int          `json:"nodeCount"`
-	CPUSum         int          `json:"cpuSum"`
-	CloudProvider  string       `json:"cloudProvider"`
-	HelmVersion    string       `json:"helmVersion"`
-	ClusterVersion string       `json:"clusterVersion"`
-	LastReportTime *time.Time   `json:"lastReportTime,omitempty"`
-	LastKeepAlive  *time.Time   `json:"lastKeepAlive,omitempty"`
-	CreatedAt      *time.Time   `json:"createdAt,omitempty"`
-	IsConnected    bool         `json:"isConnected"`
-	Capabilities   []Capability `json:"capabilities,omitempty"`
-	Status         string       `json:"status,omitempty"`
-	FailedFeatures []string     `json:"failedFeatures,omitempty"`
+	Cluster          string       `json:"cluster"`
+	NodeCount        int          `json:"nodeCount"`
+	CPUSum           int          `json:"cpuSum"`
+	CloudProvider    string       `json:"cloudProvider"`
+	HelmVersion      string       `json:"helmVersion"`
+	ClusterVersion   string       `json:"clusterVersion"`
+	LastReportTime   *time.Time   `json:"lastReportTime,omitempty"`
+	LastKeepAlive    *time.Time   `json:"lastKeepAlive,omitempty"`
+	CreatedAt        *time.Time   `json:"createdAt,omitempty"`
+	IsConnected      bool         `json:"isConnected"`
+	Capabilities     []Capability `json:"capabilities,omitempty"`
+	Status           string       `json:"status,omitempty"`
+	FailedFeatures   []string     `json:"failedFeatures,omitempty"`
+	ConnectionTime   *time.Time   `json:"connectionTime,omitempty"`
```

Comment:
What is connection time?
We already have created_at (which is first connected) and last keep alive which is last connection time